share/completions/jhipster.fish

```diff
@@ -0,0 +1,37 @@
+# JHipster 4.9.0
+
+# Check if command already given
+function __fish_prog_needs_command
+    set cmd (commandline -opc)
+    echo $cmd
+    if [ (count $cmd) -eq 1 -a $cmd[1] = 'jhipster' ]
```

Comment:
Please remove the `$cmd[1] = 'jhipster'` part - that breaks if you use an alias or any other way of "wrapping" this. It's also completely unnecessary, since fish has already determined that `jhipster` should be completed.